src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -87,8 +89,8 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 				getFormat(general, media, currentAudioTrack, MI.Get(general, 0, "Format"), file);
 				getFormat(general, media, currentAudioTrack, MI.Get(general, 0, "CodecID").trim(), file);
 				media.setDuration(getDuration(MI.Get(general, 0, "Duration/String1")));
-				media.setBitrate(getBitrate(MI.Get(general, 0, "OverallBitRate")));
-				media.setTruncated(getTruncated(MI.Get(general, 0, "IsTruncated")));
+				media.setBitrate(parseBitRate(MI.Get(general, 0, "OverallBitRate")));
+				media.setTruncated(getTruncated(MI.Get(general, 0, "IsTruncated"))); //TODO: (Nad) What?
```

Comment:
@Sami32 I made that comment before you gave me the answer, and I'm letting the "bookmark" stand until I've taken care of it (parsing, storing in database).